### ISSUE TITLE
Show info header only once

### ISSUE THREAD

**Issue by ahangarha** [shakacode/heroku-to-control-plane#111]
Some commands (like `setup-app`) runs other commands. In such cases, we show the info header (for `org` and `app` name) twice.

Example:
```
➜ cpl setup-app
ORG: mostafa-demo
APP: tutorial-app

ORG: mostafa-demo
APP: tutorial-app

Applying template 'rails'...
```

**Comment by justin808** [comment]
@ahangarha any other commands? or just setup-app

**Comment by justin808** [comment]
This is really low priority.

**Comment by ahangarha** [comment]
I noticed it in `setup-app` but it should be the same for `maintenance` and `promote-app-from-upstream`.
